src/common/qprojectmconfigdialog.cpp

```diff
@@ -126,14 +126,14 @@ void QProjectMConfigDialog::saveConfig() {
     // Will only keep data_dir.
     auto settings = projectm_get_settings(_qprojectMWidget->qprojectM()->instance());
```

Comment:
Hum... `projectm_get_settings()` didn't make it into the 4.0 release, the whole settings struct is gone and was replaced by get/set calls.

Better try compiling against the 4.0 release or current master of libprojectM.

Reply:
I believe I am? See this output:

```bash
(base) carm@blackmage:~/projects/frontend-qt$ ldd src/ui-jack/projectM-jack |grep projectM
        libprojectM.so.4 => /usr/local/lib/libprojectM.so.4 (0x00007fce9bc00000)
```

Reply:
The .so version is about misleading as it's just counted up, and not really related to the version number. The filename of projectM 4 is different though, it's named `libprojectM-4.so`. The include dirs also are prefixed with a `projectM-4` path.

Just try to build the Qt app against latest projectM master, making sure no libs/headers of older versions are still in you system paths.